src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs

```diff
@@ -71,20 +71,20 @@ partial class ApplicationEngine
             [OpCode.DEPTH] = 60,
             [OpCode.DROP] = 60,
             [OpCode.NIP] = 60,
-            [OpCode.XDROP] = 400,
-            [OpCode.CLEAR] = 400,
-            [OpCode.DUP] = 60,
-            [OpCode.OVER] = 60,
-            [OpCode.PICK] = 60,
-            [OpCode.TUCK] = 60,
+            [OpCode.XDROP] = 4000,
+            [OpCode.CLEAR] = 2000,
+            [OpCode.DUP] = 4000,
```

Comment:
Why `DUP` cost so much? Can you provide the test cases?

Reply:
Some changes are from nspcc's result, but it seems that the vm logic of neo C# and neo-go is somehow different, I reverted these changes.